venia-integration-tests/src/assertions/wishlist/index.js

```diff
@@ -43,3 +46,21 @@ export const assertProductInWishlist = productName => {
         .children(wishlistItemAddToCartButton)
         .should('exist');
 };
+
+/**
+ * Utility function to assert create wishlist link exists
+ */
+export const assertCreateWishlistLink = () => {
+    // assert create a list link exists
```

Comment:
Consider dropping this comment. You've already captured it in the function name and the function block comment.